### PR TITLE
aave v3 rewards claim | Avalanche

### PR DESCRIPTION
**Pull request by pradyuman-verma**
- Avax aave v3 rewards claim
- [x] review and approve connector @thrilok209 @KaymasJain 
- [x] deploy and comment address @pradyuman-verma 
- [x] enable and add to SDK

### CODE REVIEW

**Comment by pradyuman-verma** [comment]
### Deployed on Avax

**Name** `AAVE-V3-CLAIM-A`
**Addr** [0x9341dd1191837C3ab30447ADa907F7625a61CF14](https://snowtrace.io/address/0x9341dd1191837C3ab30447ADa907F7625a61CF14#code)
**tx** [here](https://snowtrace.io/tx/0x19d91a841c3f63b5c9d8cf0168cb04a1cbc8891df7eae4ce7f2c1ebe19b08abe)

File: contracts/avalanche/connectors/aave/aave-rewards/v3/main.sol [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	function claimAll(address[] calldata assets)
+		external
+		payable
+		returns (string memory _eventName, bytes memory _eventParam)
+	{
+		require(assets.length > 0, "invalid-assets");
+		uint256[] memory _amt = new uint256[](assets.length);
+		address[] memory _rewards = new address[](assets.length);
+
+		(_rewards, _amt) = incentives.claimAllRewards(assets, address(this));
```
Comment (thrilok209) [inline contracts/avalanche/connectors/aave/aave-rewards/v3/main.sol:65]: @pradyuman-verma we can add claimAll function which won't any assets params also, right?

Comment (pradyuman-verma) [reply contracts/avalanche/connectors/aave/aave-rewards/v3/main.sol:65]: No claim all takes asset as a parameter.

Comment (pradyuman-verma) [reply contracts/avalanche/connectors/aave/aave-rewards/v3/main.sol:65]: Also, it is better to pass the asset address as a parameter.

**Review by thrilok209 — APPROVED** [review]

**Pull request merged**